app/src/androidTest/java/org/simple/clinic/di/TestSyncConfigModule.kt

```diff
@@ -7,15 +7,26 @@ import org.simple.clinic.sync.SyncConfigType
 import org.simple.clinic.sync.SyncConfigType.Type.Frequent
 import org.simple.clinic.sync.SyncGroup
 import org.simple.clinic.sync.SyncInterval
+import java.time.Duration
 
 @Module
 class TestSyncConfigModule {
 
+  @Provides
+  fun syncInterval(): SyncInterval {
+    return SyncInterval(
+        frequency = Duration.ofMinutes(16),
+        backOffDelay = Duration.ofMinutes(5L)
```

Comment:
What is 5L minutes?

Reply:
Ah the `L` suffix means the type is a `Long`. The `ofMinutes` method takes a `Long` as a parameter.